src/test/java/com/wikia/webdriver/testcases/signuptests/SignUpTests.java

```diff
@@ -194,6 +194,38 @@ public void SignUp_007_signUpWithFacebook() {
     base.logOut(wikiURL);
   }
 
+  @Test(groups = {"SignUp_008", "SignUp"})
+  public void SignUp_008_signupJapaneseUser() {
+    String wikiURLJapanese = "http://ja.ja-test.wikia.com/";
+    WikiBasePageObject base = new WikiBasePageObject(driver);
+    SignUpPageObject signUp = base.navigateToSpecialSignUpPage(wikiURLJapanese);
+    signUp.disableCaptcha();
+    String userName = "品質管理" + signUp.getTimeStamp();
+    String password = "品質管理管理" + signUp.getTimeStamp();
+    String email = credentials.emailQaart2;
+    String emailPassword = credentials.emailPasswordQaart2;
+
+    signUp.typeEmail(email);
+    signUp.typeUserName(userName);
+    signUp.typePassword(password);
+    signUp.enterBirthDate(PageContent.WIKI_SIGN_UP_BIRTHMONTH, PageContent.WIKI_SIGN_UP_BIRTHDAY,
+            PageContent.WIKI_SIGN_UP_BIRTHYEAR);
+    AlmostTherePageObject almostTherePage = signUp.submit(email, emailPassword);
+    //almostTherePage.verifyAlmostTherePage();
```

Comment:
If you comment something please also write why you decided to leve it, otherwise please remove that line